gradle.properties

```diff
@@ -1,3 +1,4 @@
 ideaVersion = IC-15.0.2
 javaVersion = 1.6
 ijPluginRepoChannel = alpha
+version = 0.9.1-alpha-SNAPSHOT
```

Comment:
Not yet!

We need to package into the plugin resources similarly to how we include
the usageTrackerProperty.


On Thu, Jan 7, 2016 at 1:45 PM, Elliotte Rusty Harold <
notifications@github.com> wrote:

> In gradle.properties
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/341#discussion_r49109122>
> :
>
> > @@ -1,3 +1,4 @@
> >  ideaVersion = IC-15.0.2
> >  javaVersion = 1.6
> >  ijPluginRepoChannel = alpha
> > +version = 0.9.1-alpha-SNAPSHOT
>
> Any obvious way to reference this from our Java code? e.g. for User-Agent
> string?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/gcloud-intellij/pull/341/files#r49109122>
> .
>